components/gcp/dataflow/launch_python/README.md

```diff
@@ -59,83 +76,60 @@ The component does several things during the execution:
-In this sample, we run a wordcount sample code in a Kubeflow Pipeline. The output will be stored in a Cloud Storage bucket. Here is the sample code:
-
+The following sample code works in an IPython notebook or directly in Python code. See the sample code below to learn how to execute the template.
+In this sample, we run a wordcount sample code in a Kubeflow pipeline. The output will be stored in a Cloud Storage bucket. Here is the sample code:
 
 ```python
 !gsutil cat gs://ml-pipeline-playground/samples/dataflow/wc/wc.py
 ```
 
-    #
-    # Licensed to the Apache Software Foundation (ASF) under one or more
-    # contributor license agreements.  See the NOTICE file distributed with
-    # this work for additional information regarding copyright ownership.
-    # The ASF licenses this file to You under the Apache License, Version 2.0
-    # (the "License"); you may not use this file except in compliance with
-    # the License.  You may obtain a copy of the License at
-    #
+<!--is this licenese required here, no other compnenet has the license here, i think we can remove it-->
+     # Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.  See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+    # The ASF licenses this file to You under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
     #    http://www.apache.org/licenses/LICENSE-2.0
+    # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
     #
-    # Unless required by applicable law or agreed to in writing, software
-    # distributed under the License is distributed on an "AS IS" BASIS,
-    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    # See the License for the specific language governing permissions and
-    # limitations under the License.
-    #
-    
-    """A minimalist word-counting workflow that counts words in Shakespeare.
-    
-    This is the first in a series of successively more detailed 'word count'
-    examples.
-    
-    Next, see the wordcount pipeline, then the wordcount_debugging pipeline, for
-    more detailed examples that introduce additional concepts.
+  
+ #### A minimalist word-counting workflow that counts words in Shakespeare 
+This is the first in a series of successively more detailed word count examples. Next, see the wordcount pipeline, then the wordcount_debugging pipeline, for more detailed examples that introduce additional concepts. <!--where should the user find this examples, can we give a link to these?-->
```

Comment:
where can users find the word count pipeline and wordcount debugging pipeline? is there a link to these?